### PR TITLE
Multi page manual

### PR DESCRIPTION
**Pull request by lucacasonato**
Moved to multi page manual as discussed offline. Also added some extra text to tie things together.

### CODE REVIEW

File: docs/getting_started/first_steps.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+no different to one you can run in the browser:
+
+```typescript
+console.log("Welcome to Deno 🦕");
+```
+
+Try the program:
+
+```bash
+deno https://deno.land/std/examples/welcome.ts
```
Comment (nayeemrmn) [inline docs/getting_started/first_steps.md:27]:
```suggestion
deno run https://deno.land/std/examples/welcome.ts
```

**Comment by ry** [comment]
please merge with master

**Comment by ry** [comment]
there's a flaky test that will be removed soon (https://github.com/denoland/deno/pull/5116) I'll land this afterwards

**Comment by machadolucasvp** [comment]
Hello!, I'm sorry but the hyperlink to styles_guide added by this commit doesn't exists, is intentional, do you plan to add it later?

**Pull request merged**